src/components/appointment/AppointmentDetail.tsx

```diff
@@ -19,20 +19,27 @@ const AppointmentDetail: React.FC<AppointmentDetailProps> = ({ appointment, cour
     const toast = useToast();
     const { space, sizes } = useTheme();
     const [canceled, setCanceled] = useState<boolean>(false);
+    const { date, startTime, endTime } = getAppointmentDateTime(appointment.start, appointment.duration);
+
+    const countAttendees = useCallback(() => {
+        // TODO change to appointment.appointment_participant_pupil, appointment.appointment_participant_student, appointment.appointment_participant_screener
+        const participants = appointment.participants ? appointment.participants.length : 0;
+        const organizers = appointment.organizers ? appointment.organizers.length : 0;
+        return participants + organizers;
+    }, [appointment.organizers, appointment.participants]);
+    const attendeesCount = useMemo(() => countAttendees(), [countAttendees]);
```

Comment:
I guess these are the right dependencies
```suggestion
    const attendeesCount = useMemo(() => countAttendees(), [appointment.organizers, appointment.participants]);
```